packages/connectivity/connectivity_macos/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: connectivity_macos
+description: macOS implementation of the connectivity plugin.
+version: 0.0.1
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/connectivity/connectivity_macos
+
+flutter:
+  plugin:
+    platforms:
+      macos:
+        pluginClass: ConnectivityPlugin
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+  flutter: ">=1.13.0 <2.0.0"
```

Comment:
How did the web federated implementations land then?

Reply:
Uh, good question. I didn't realize that they were requiring above stable and I would also expect them to break CI in the same way this PR is currently red. I'll do some digging.

Reply:
@franciscojma86 explained this to me offline. So our [`.cirrus.yml` actually _deletes_ the web files off of the local CI checkout](https://github.com/flutter/plugins/blob/master/.cirrus.yml#L34) before CI runs to avoid this issue. We have an outstanding issue to remove that hack once the next stable rolls. flutter/flutter#42864 We can't use that approach here easily but Francisco is going to just add in a dummy Android project to the macOS implementation for now in order to make this compatible with the current stable instead. Once v1.13 rolls to the next stable he'll take it out of the macOS implementation and up the version constraint.